src/pages/groupComparison/MultipleCategoryBarPlot.tsx

```diff
@@ -424,12 +435,56 @@ export default class MultipleCategoryBarPlot extends React.Component<
     }
 
     @computed get labels() {
+        if (this.props.sortOption == 'sortByCount') {
+            _.forEach(this.data, item => {
+                item.counts.sort((a, b) =>
+                    a.majorCategory.localeCompare(b.majorCategory)
+                );
+            });
+            const totalSumArray: TotalSumItem[] = [];
```

Comment:
this code is quite confusing.   it would be clearer if you used _.sortBy(collection, ()=>);   where the return of the callback is a _.sum of the item counts within.

Reply:
The logic is that I will traverse this.data and will make one totalSumArray array which has a schema like:
```
{
    majorCategory: countItem.majorCategory,
    sum: countItem.count,
    minorCategory: [
        {
            name: item.minorCategory,
            count: countItem.count,
            percentage: countItem.percentage,
        },
    ],
}

```

Now, I will traverse this.data and go to each `majorCategory` to update the `totalSum` in which i will store the count of each object. Then, I will sort the `totalSumArray` with respect to `totalSum` and extract the information as per the requirements like label and `majorCategory`.

Let's take an example. For one configuration, this.data contains:
```
{
    0: { minorCategory: 'Male', counts: Array(42) },
    1: { minorCategory: 'Female', counts: Array(42) }
}
```
I will traverse each object, go to the counts array, and create one object for like Melanoma in `totalSumArray` in which I will store the information and sum of counts for each object. Then, I will sort the `totalSumArray` according to sum and retrieve the labels from there.